changelog

```diff
@@ -1,3 +1,202 @@
+version 0.10.7
+  * Big changes:
+    * Rebalanced ship mass and engine performance across the game. (@Quantumshark)
+      * Increased thrust, turn, reverse thrust, and afterburner thrust by 50% for all engines.
+    * Disabled fighters and drones will no longer be hit by stray projectiles. (@tibetiroka)
+    * Reverted the changes to the acceleration and initial velocity of human missiles made in 0.10.0. (@Anarchist2)
+    * Added many new uninhabited planets and stations to Human space. (@bene-dictator, @Quantumshark)
+    * Turret hardpoint can have limited firing arcs. (@1010todd)
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@a4358, @Arachi-Lover, @bene-dictator, @enot888, @tibetiroka, @TomGoodIdea)
+      * Earth day jobs no longer offer unless you can land on Earth. (@ziproot)
+      * Make TMBR 3a dialog choice order consistent. (@ziproot)
+      * Add blocked message to Hai Film Crew 3. (@neurotrope)
+      * Add missing indent, remove unnecessary labels and gotos to Mafia Extortion mission. (@Arachi-Lover)
+      * Fix overlapping stars in Umbral system. (@Saugia)
+      * Fix attribute on Vyrmeid lifeform. (@ziproot)
+      * Give Cruiser in "FWC Cebalrai 1B" Combat Drones instead of Lances. (@Quantumshark)
+      * Add new "CCOR Logistics" fleet for non-combat ships. (@bene-dictator)
+      * Allow quantum keystone jobs to offer properly. (@ziproot)
+      * Update New China and Geyser to have outfitters and tribute fleets that match their region. (@bene-dictator)
+      * Fix the attribution of various copyright entries. (@warp-core)
+      * Remove duplicate Korath phrase. (@tibetiroka)
+      * Clarify the description of the Far Osk 27. (@ziproot)
+      * Remove outfit scanning capabilities from Grey's person ship. (@mOctave)
+      * Block "Lunarium: Questions" from completing while assisting the Heliarchs. (@ziproot)
+      * Reword Deep's description. (@roadrunner56)
+      * List correct attributions for Unsplash images. (@ziproot)
+      * Add back instances of "ser" and explain what it means in the Skadenga and Stones of Our Fathers missions. (@bene-dictator)
```

Comment:
```suggestion
      * Made TMBR 3a dialog choice order consistent. (@ziproot)
      * Added blocked message to Hai Film Crew 3. (@neurotrope)
      * Added missing indent, remove unnecessary labels and gotos to Mafia Extortion mission. (@Arachi-Lover)
      * Fixed overlapping stars in Umbral system. (@Saugia)
      * Fixed attribute on Vyrmeid lifeform. (@ziproot)
      * Gave Cruiser in "FWC Cebalrai 1B" Combat Drones instead of Lances. (@Quantumshark)
      * Added new "CCOR Logistics" fleet for non-combat ships. (@bene-dictator)
      * Allowed quantum keystone jobs to offer properly. (@ziproot)
      * Updated New China and Geyser to have outfitters and tribute fleets that match their region. (@bene-dictator)
      * Fixed the attribution of various copyright entries. (@warp-core)
      * Removed duplicate Korath phrase. (@tibetiroka)
      * Clarified the description of the Far Osk 27. (@ziproot)
      * Removed outfit scanning capabilities from Grey's person ship. (@mOctave)
      * Blocked "Lunarium: Questions" from completing while assisting the Heliarchs. (@ziproot)
      * Reworded Deep's description. (@roadrunner56)
      * Listed correct attributions for Unsplash images. (@ziproot)
      * Added back instances of "ser" and explained what it means in the Skadenga and Stones of Our Fathers missions. (@bene-dictator)
```